pom.xml

```diff
@@ -90,9 +90,4 @@
     </site>
   </distributionManagement>
 
-  <properties>
-    <jenkins.version>1.580.1</jenkins.version>
```

Comment:
Well, it is fine to override them. Parent POM has been intentionally designed to support flexible management of Jenkins core dependencies by plugin maintainers.

On this way one of the potential issues is [Plugin POM 2.30](https://github.com/jenkinsci/plugin-pom/blob/master/CHANGELOG.md#230), which bumps Jetty and hence requires Java 8 for building & testing plugins (though they can still be compatible with Java 7). But before that the same parent pom should work well for 1.609.1+ core versions. Older versions can be also used, but it may require additional patches for HTML Unit (custom Jenkins Test Harness version).